index.html

```diff
@@ -4444,13 +4444,7 @@ <h2 id="AudioBufferSourceNode">
               between 0 and the duration of the buffer. If
               <code>loopStart</code> is less than 0, looping will begin at 0.
               If <code>loopStart</code> is greater than the duration of the
-              buffer, looping will begin at the end of the buffer. This
-              attribute is converted to an exact sample frame offset within the
-              buffer by multiplying by the buffer's sample rate and rounding to
-              the nearest integer value. Thus its behavior is independent of
-              the value of the <a href=
```

Comment:
Seconding @rtoy's observation that we have to specify that the loop points (though now unquantized) are defined in terms of the *buffer's* sample rate, not the computed playback rate.

If loop points were evaluated in terms of computed playback rate, the behavior with a modulated playback rate would be chaotic and unpredictable.